core/AutocorrectSuggestion.h

```diff
@@ -6,10 +6,15 @@
 namespace sorbet::core {
 
 struct AutocorrectSuggestion {
-    core::Loc loc;
-    std::string replacement;
+    struct Edit {
+        core::Loc loc;
```

Comment:
can these fields be `const`?
ditto for fields of `AutocorrectSuggestion` while you're still here :-)

Reply:
alas, only one could be without LLVM yelling at me (will investigate more later). `edits` definitely can't